tests/main/views/test_drafts.py

```diff
@@ -435,6 +437,8 @@ def test_should_only_copy_published_services_from_correct_framework_and_lot(self
             Service.framework_id == 1,  # G-Cloud 6
             Service.lot_id == 2,
             Service.status == 'published'
+        ).order_by(
+            desc(Service.data['serviceName'].astext)
```

Comment:
You sure this is "unique" enough?

Reply:
The service names are generated from the service's ID so should be.